src/coreclr/pal/inc/rt/intsafe.h

```diff
@@ -0,0 +1,1401 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I am confused by github showing this file being added while it was in the repo before.

Reply:
Hi @janvorli , I have checked this file was removed in below commit on 12-June-2022 (3 days back.) https://github.com/dotnet/runtime/commit/27195f670937c7e21ab68a806396f9d17c57231a

I have also deleted intsafe.h file from my branch. And commit the change.